src/TypesService.ts

```diff
@@ -3,7 +3,7 @@ import { Dependency } from "./PackageWatcher";
 import { InstallCallback, UninstallCallback } from "./shared";
 
 export class TypingsService {
-    constructor(private rootPath: string) { }
+    constructor(private rootPath: string, private useYarn: boolean) { }
```

Comment:
Would it be better to pass this in as a variable like we're using for `isDev` on line 67? Only reason why I ask is because setting it here means that if the user changes the preference they'll need to restart visual studio code for it to take effect as TypingsService will need to be reinitialised. Whereas if we pass the value as a variable to `installCommand` they can change the preference whenever they want.